src/main/java/se/sundsvall/messagingsettings/integration/db/entity/MessagingSettingsEntity.java

```diff
@@ -0,0 +1,80 @@
+package se.sundsvall.messagingsettings.integration.db.entity;
+
+import static org.hibernate.Length.LONG;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+import java.time.OffsetDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.TimeZoneStorage;
+import org.hibernate.annotations.TimeZoneStorageType;
+import org.hibernate.annotations.UpdateTimestamp;
+import se.sundsvall.messagingsettings.integration.db.entity.enums.SnailMailMethod;
+
+@Entity
+@Table(name = "messaging_settings", indexes = {
+	@Index(name = "idx_messaging_settings_municipality_id_department_id", columnList = "municipality_id, department_id"),
+})
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+public class MessagingSettingsEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Consider using UUID as identifier to make int harder for possible malicious users to guess id of entities (as I guess the service will have some kind of create/update-resource which possibly will use the id of the entity)? I.e. change to:
```
	@Id
	@GeneratedValue(strategy = GenerationType.UUID)
	@Column(name = "id")
	private String id;
```